install/deb/apache2/apache2.conf

```diff
@@ -17,15 +17,6 @@ KeepAlive Off
 MaxKeepAliveRequests 100
 KeepAliveTimeout 10
 
-<IfModule mpm_prefork_module>
```

Comment:
If a user ever need to to restore the apache2.conf because the *** something up we should leave this part intact 

For now just add an Deprecation notice and leave it here.

Reply:
I was thinking, would it also be an idea to show a deprecation message at the ui fields that the functionality is going to be removed and that the should switch over to php-fpm?